calico/felix/felix.py

```diff
@@ -245,7 +245,7 @@ def dump_top_level_actors(log):
 
         # Count cluster existence
         if config.USAGE_COUNT:
```

Comment:
Why not remove this one-off call to `usage_count()` now you have a greenlet to do it?  This call will always report a felix_count of 0 and the greenlet should be started within a second or two so we're not really gaining much by doing an extra log here.